profilingdemo/source/c/src/kaa_demo.c

```diff
@@ -0,0 +1,151 @@
+/**
+ *  Copyright 2014-2016 CyberVision, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+#include <stdio.h>
+#include <time.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/stat.h>
+#include <unistd.h>
+
+#include <kaa_error.h>
+#include <platform/kaa_client.h>
+#include <kaa_configuration_manager.h>
+#include <kaa_profile.h>
+#include <platform/ext_key_utils.h>
+
+#define demo_printf(msg, ...) printf((msg), ##__VA_ARGS__)
```

Comment:
The `demo_printf` macro is defined in `common/c/targets/...`.
If you prefer to use it (and if demo is planned to be run on non-posix platforms), include the `common/c` directory into the demo.